packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -343,15 +345,23 @@ const ReplayChart = observer(({ is_accumulator_contract }) => {
                         is_bottom_widget_visible={isBottomWidgetVisible()}
                     />
                 ))}
-            {!is_beta_chart && is_accumulator_contract && !!markers_array && (
+            {!is_beta_chart && is_accumulator_contract && !!markers_array && !is_reset_contract && (
                 <DelayedAccuBarriersMarker
-                    marker_component={accu_barriers_marker_component}
-                    key={accumulators_barriers_marker.key}
+                    marker_component={MarkerComponent}
+                    key={barriers_marker.key}
                     is_dark_theme={is_dark_theme}
                     granularity={granularity}
                     is_in_contract_details
                     previous_spot_time={accumulator_previous_spot_time}
-                    {...accumulators_barriers_marker}
+                    {...barriers_marker}
+                />
+            )}
+            {is_reset_contract && !!markers_array && reset_time && (
```

Comment:
i think it won't work for `is_beta_chart`, please check with @balakrishna-deriv 
we might need to do parallel implementation of this in flutter-charts..
 also, it's better to put these long conditions outside in variables:
`const should_show_accumulators_barrier_range = !is_beta_chart && is_accumulator_contract;`
`const should_show_reset_barrier_shade = !is_beta_chart && is_reset_contract && !!markers_array && reset_time;`

Reply:
I've already inform @balakrishna-deriv

Reply:
Yes, I'll work with @kate-deriv on this :)